Source/Core/Constants+Style.swift

```diff
@@ -0,0 +1,51 @@
+
+extension Constants {
+    
+    enum Style {
+
+        enum BillingForm {
+
+            enum CancelButton: Double {
+                case height = 44
+                case width = 53
+                case fontSize = 17
+            }
+
+            enum DoneButton: Double {
+                case height = 44
+                case width = 53
+                case fontSize = 17
+            }
+
+            enum HeaderTitle: Double {
+                case fontSize = 24
+            }
+
+            enum InputErrorLabel: Double {
+                case height = 18
+                case fontSize = 13
+            }
+
+            enum InputHintLabel: Double {
+                case fontSize = 13
+            }
+
+            enum InputTextField: Double {
+                case height = 56
+                case width = 335.0
```

Comment:
I am assuming these are default values for width and height and merchants can override them if needed later. But what about different screen sizes for iPhone and iPad do you want hardcode the width